### DISCUSSION TITLE
Error in setting up container

### DISCUSSION THREAD

**Discussion by amritesh19** [owid/owid-grapher#1373] [Q&A]
Hi, I am trying to setup development environment within VS Code devcontainer, but encountering this error. 

![image](https://user-images.githubusercontent.com/73696688/162587109-78a59100-163d-4118-a8cd-d05438e3cbc2.png)
![image](https://user-images.githubusercontent.com/73696688/162587165-64476cc3-c1e3-485c-8fbb-3385cb2275f1.png)

Please help me resolve this.

**Comment by jcdevilleres** [comment]
Restart or reinstall Docker Engine
